test/mock-connector.py

```diff
@@ -20,6 +20,7 @@
 
 import pika
 import logging
+import json
```

Comment:
About commit message 76787b3d84e4d4291318563074cbdddf2fd8fa07:
In title say that it is related to mock-connector